src/components/Product/Product.module.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../styles/abstracts/variables';
```

Comment:
I reckon this file should go in `styles/pages/` - the goal is that non-component specific styling goes there